examples/min-platform/src/main.rs

```diff
@@ -76,6 +76,19 @@ fn main() -> Result<()> {
         config.signals_based_traps(false);
     }
 
+    // For x86_64 targets be sure to enable relevant CPU features to avoid
+    // float-related libcalls which is required for the `x86_64-unknown-none`
+    // target.
+    if cfg!(target_arch = "x86_64") {
+        unsafe {
+            config.cranelift_flag_enable("has_sse3");
+            config.cranelift_flag_enable("has_ssse3");
+            config.cranelift_flag_enable("has_sse41");
+            config.cranelift_flag_enable("has_sse42");
+            config.cranelift_flag_enable("has_fma");
```

Comment:
Looking this up to verify -- this has existed since 2013 (Haswell) on Intel chips and 2012 (Piledriver) on AMD chips so it's probably fine for anyone wanting to actually run the example on a semi-modern dev machine or server. Perhaps we want to put a comment here ("Note: these features should be present on Intel Haswell (2013) / AMD Piledriver (2012) or later; make sure to run this example on such a platform.") next to the config?